BibBuddy/app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,13 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="de.bibbuddy">
 
+    <uses-permission android:name="android.permission.CAMERA" />
+    <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
I think we have a merge conflict here due to the overlap of this PR and the API - should be resolved.